playbooks/openstack/configuration.md

```diff
@@ -778,6 +780,55 @@ enabled_handlers = vif,lb,lbaasspec,namespace,*kuryrnet*
 Note depending on the Kuryr controller image version, this feature (handler)
 may not be available.
 
+The `kuryr_openstack_global_namespace` defines the list of namespaces that are
+global and can be accessed from any other namespace, as well as access any
+namespace. By default only the `default` and the `openshift-monitoring`
+namespaces are included in the list.
```

Comment:
I would also like to see openshift-logging, if possible.

Reply:
you can add openshift-logging with this option in the inventory. It seems openshift-logging is not installed by default so we should not make it the default either